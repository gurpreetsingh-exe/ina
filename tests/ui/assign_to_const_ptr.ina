// COMPILE_FAIL

fn main() {
    let a = 20;
    let b = &a as *i32;
    *b = 50;
}

