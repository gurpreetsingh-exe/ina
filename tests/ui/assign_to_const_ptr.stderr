error: cannot assign to `*b`, which is behind a `*` pointer
   ╭─[tests/ui/assign_to_const_ptr.ina:6:5]
   │
 6 │     *b = 50;
   │     ╶╮
   │      ╰── cannot assign to this
   ∙
───╯
